opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -521,11 +497,14 @@ public SELF user_selected_sca_challenge_type_email1_to_embedded_authorization()
         return self();
     }
 
-    public SELF user_max_musterman_provided_correct_sca_challenge_result_after_wrong_to_embedded_authorization_and_sees_redirect_to_fintech_ok() {
-        assertThat(this.redirectUriToGetUserParams).contains("sca-result").contains("/EMAIL").contains("wrong=true");
+    public void user_max_musterman_provided_correct_sca_challenge_result_after_wrong_to_embedded_authorization_and_sees_redirect_to_fintech_ok() {
```

Comment:
Please return `SELF` as it makes this step `continuable` within `AccountInformationRequestCommon` class

Reply:
done